lang/crystal/Portfile

```diff
@@ -2,10 +2,9 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
-PortGroup           openssl 1.0
```

Comment:
Does Crystal no longer use OpenSSL?

Reply:
It does, good catch, I forgot to add the `depends` line. No need for the PortGroup.